notes/stv/20230125-red-deploy-01.txt

```diff
@@ -0,0 +1,952 @@
+        source /deployments/hadoop-yarn/bin/deploy.sh
+        
+        > 
+        TASK [Linking data directories] ************************************************
+fatal: [zeppelin]: FAILED! => {"msg": "Invalid data passed to 'loop', it requires a list, got this instead: None. Hint: If you passed a list/dict of just one element, try adding wantlist=True to your lookup invocation or use q/query instead of lookup."}
+fatal: [worker01]: FAILED! => {"msg": "Invalid data passed to 'loop', it requires a list, got this instead: None. Hint: If you passed a list/dict of just one element, try adding wantlist=True to your lookup invocation or use q/query instead of lookup."}
+fatal: [worker02]: FAILED! => {"msg": "Invalid data passed to 'loop', it requires a list, got this instead: None. Hint: If you passed a list/dict of just one element, try adding wantlist=True to your lookup invocation or use q/query instead of lookup."}
+fatal: [worker03]: FAILED! => {"msg": "Invalid data passed to 'loop', it requires a list, got this instead: None. Hint: If you passed a list/dict of just one element, try adding wantlist=True to your lookup invocation or use q/query instead of lookup."}
+fatal: [worker04]: FAILED! => {"msg": "Invalid data passed to 'loop', it requires a list, got this instead: None. Hint: If you passed a list/dict of just one element, try adding wantlist=True to your lookup invocation or use q/query instead of lookup."}
+fatal: [worker05]: FAILED! => {"msg": "Invalid data passed to 'loop', it requires a list, got this instead: None. Hint: If you passed a list/dict of just one element, try adding wantlist=True to your lookup invocation or use q/query instead of lookup."}
+fatal: [worker06]: FAILED! => {"msg": "Invalid data passed to 'loop', it requires a list, got this instead: None. Hint: If you passed a list/dict of just one element, try adding wantlist=True to your lookup invocation or use q/query instead of lookup."}
+
+
+
+	aglais:
+	  status:
+	    deployment:
+	      type: hadoop-yarn
+	      conf: zeppelin-54.86-spark-6.26.43
+	      name: iris-gaia-red-20230125
+	      date: 20230125T123051
+	      hostname: zeppelin.gaia-dmp.uk
+	  spec:
+	    openstack:
+	      cloud:
+		base: arcus
+		name: iris-gaia-red
+
+	real	56m34.042s
+	user	10m18.838s
+	sys	2m44.551s
+
+
+	
+# -----------------------------------------------------
+# Allow port 8080 (HTTP).
+# TODO Fix the HTTPS proxy to work on all deployments.
+# https://github.com/wfau/gaia-dmp/issues/838
+#[root@ansibler]
+
+    groupid=$(
+        openstack \
+            --os-cloud "${cloudname:?}" \
+            security group list \
+                --format json \
+        | jq -r '.[] | select(.Name | match("zeppelin-security")) | .ID'
+        )
+
+    openstack \
+        --os-cloud "${cloudname:?}" \
+        security group rule create \
+            --ingress \
+            --protocol 'tcp' \
+            --dst-port '8080:8080' \
+            "${groupid}"
+
+    >   ....
+    >   ....
```

Comment:
Closed port  8080 for this deploy
https://github.com/stvoutsin/gaia-dmp/blob/feature/202301-red-deploy/notes/stv/20230127-delete-8080-rule-red.txt
Since you've added the issues I'll set this to resolved